routers/install/routes.go

```diff
@@ -15,12 +15,20 @@ import (
 	"code.gitea.io/gitea/modules/templates"
 	"code.gitea.io/gitea/modules/web"
 	"code.gitea.io/gitea/modules/web/middleware"
-	"code.gitea.io/gitea/routers"
+	"code.gitea.io/gitea/routers/common"
 	"code.gitea.io/gitea/services/forms"
 
 	"gitea.com/go-chi/session"
 )
 
+type dataStore struct {
+	Data map[string]interface{}
+}
+
+func (d *dataStore) GetData() map[string]interface{} {
+	return d.Data
+}
+
```

Comment:
Not necessarily for this PR but could we not use:

```go
type dataStore map[string]interface{}

func (d *dataStore) GetData() map[string]interface{} {
   return *d
}
```

with attendant changes.